packit_service/fedmsg/consumer.py

```diff
@@ -0,0 +1,67 @@
+# MIT License
+#
+# Copyright (c) 2018-2019 Red Hat, Inc.
+
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+import logging
+
+from fedora_messaging import api, config
+from fedora_messaging.message import Message
+
+config.conf.setup_logging()
+logger = logging.getLogger(__name__)
+
+
+class Consumerino:
+    """
+    Consume events from fedora messaging
+    """
+
+    @staticmethod
+    def fedora_messaging_callback(message: Message):
+        print(f"{message.topic}: {message.body}")
+
+    @staticmethod
+    def consume_from_fedora_messaging():
+        """
+        fedora-messaging is written in an async way: callbacks
+        """
+        queue_name = "708D1D74-63E4-472A-88E8-8E43C5AE40DC"
```

Comment:
huh, what?

Reply:
from [docs](https://fedora-messaging.readthedocs.io/en/latest/quick-start.html#fedora-s-public-broker):

```
# Queue names *must* be in the normal UUID format: run "uuidgen" and use the
# output as your queue name. If your queue is not exclusive, anyone can connect
# and consume from it, causing you to miss messages, so do not share your queue
# name. Any queues that are not auto-deleted on disconnect are garbage-collected
# after approximately one hour.
```

Reply:
Please add a comment about who generated the UUID.

> do not share your queue name

Do we care? Shouldn't it be a secret then?

Reply:
secret +1

Reply:
@rpitonak ok, so it's in secrets now, great. But I still don't have an answer to my question about what/where/how was the UUID generated? Or where can I see this queue? Or basically, where should I go when something goes wrong?

Reply:
https://gitlab.cee.redhat.com/user-cont/packit-service/issues/8